tasks/pipeline.py

```diff
@@ -332,6 +332,50 @@ def generate_failure_messages(base):
     return messages_to_send
 
 
+@task
+def trigger_child_pipeline(_, git_ref, project_name, variables):
+    """
+    Trigger a child pipeline on a target repository and ref.
+    Used in CI jobs only (requires CI_JOB_TOKEN).
+
+    Use --variables to specify the environment variables that should be passed to the child pipeline.
+
+    Examples:
+    inv pipeline.trigger-child-pipeline --git-ref "master" --project-name "DataDog/agent-release-management" --variables "RELEASE_VERSION"
+    """
+
+    if not os.environ.get('CI_JOB_TOKEN'):
+        raise Exit("CI_JOB_TOKEN variable needed to create child pipelines.", 1)
+
+    # The Gitlab lib requires `GITLAB_TOKEN` to be set, though
+    # we won't use it here
+    os.environ["GITLAB_TOKEN"] = os.environ['CI_JOB_TOKEN']
+
+    gitlab = Gitlab()
+
+    data = {"token": os.environ['CI_JOB_TOKEN'], "ref": git_ref, "variables": {}}
+
+    # Fill the environment variables to pass to the child pipeline.
+    for v in variables.split(','):
+        data['variables'][v] = os.environ[v]
+
+    print(
+        "Creating child pipeline in repo {}, on git ref {} with params: {}".format(
+            project_name, git_ref, data['variables']
+        )
+    )
+
+    res = gitlab.trigger_pipeline(project_name, data)
+
+    if not res.get('id'):
```

Comment:
Any reason not to do this?
```suggestion
    if 'id' not in res:
```
(The suggestion is more idiomatic, although if `id` can be present but empty it doesn't make sense to make the change)